### DISCUSSION TITLE
3 Level drop down menu.

### DISCUSSION THREAD

**Discussion by ritchiemarkov** [gregsullivan/_tw#51] [Q&A]
Hello Greg!
I wonder if you have any prebuild 3 level dropdown menu navwalker.
Im trying to recreate: https://blog.logrocket.com/how-create-multilevel-dropdown-menu-react

Bootstrap has its own, but tailwind is bit different i guess.

**Comment by gregsullivan** [comment]
Hi Richard!

I'm sorry to say I don't have anything prebuilt. The one in your example looks like it could largely be recreated with CSS and hover states; my advice would be to do as much as possible that way. You can avoid building a custom nav walker by using `@apply` to style the existing classes on navigation elements. (The only outstanding piece of documentation for _tw covers some approaches to this kind of problem, but I don't expect to have time to write it until later this month.)

**Comment by ritchiemarkov** [comment]
Ty so much for your answer. I'm really at basic things still. So I'm
decided to document my self from wp and php side pov.

Really appreciate your answer.


Il Lun 8 Mag 2023, 18:36 Greg Sullivan ***@***.***> ha
scritto:

**Reply by gregsullivan** [reply]
No problem at all—happy to answer any other questions you might have!
